src/ui/include/mainwindow.h

```diff
@@ -256,16 +254,13 @@ private slots:
     int                 saveAs(EditorTabWidget *tabWidget, int tab, bool copy);
     QUrl                getSaveDialogDefaultFileName(EditorTabWidget *tabWidget, int tab);
     void                setupLanguagesMenu();
-    void                transformSelectedText(std::function<QString (const QString &)> func);
+    void                transformSelectedText(QString (*func)(const QString&));
```

Comment:
Why that change? Isn't a `std::function` all around more convenient and more compatible?